x-pack/metricbeat/module/iis/website/website_integration_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build integration
+// +build windows
+
+package website
+
+import (
+	"testing"
+
+	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
+
+	// Register input module and metricset
+	_ "github.com/elastic/beats/v7/metricbeat/module/windows"
+	_ "github.com/elastic/beats/v7/metricbeat/module/windows/perfmon"
+)
+
+func TestData(t *testing.T) {
+	metricSet := mbtest.NewReportingMetricSetV2Error(t, getConfig())
+	if err := mbtest.WriteEventsReporterV2Error(metricSet, t, "/"); err != nil {
+		// should find a way to first check if iis is running
+		//	t.Fatal("write", err)
+	}
+}
+
+func TestFetch(t *testing.T) {
+	m := mbtest.NewFetcher(t, getConfig())
+	_, errs := m.FetchEvents()
+	if len(errs) > 0 {
+		// should find a way to first check if iis is running
+		//t.Fatalf("Expected 0 error, had %d. %v\n", len(errs), errs)
```

Comment:
We might skip these tests instead of running them and ignoring errors.

Reply:
I have created a func to check for IIS on the windows machine instead.